packages/utilities/src/Async.test.ts

```diff
@@ -0,0 +1,104 @@
+import { Async, ICancelable } from './Async';
+
+describe('Async', () => {
+  describe('debounce', () => {
+    // Increase count by a specific number, to test the arguments
+    // of the debounced function;
+    let callCount = 0;
+    const fn = jest.fn((increaseCount: number) => {
+      callCount += increaseCount;
+      return callCount;
+    });
+
+    let async: Async;
+    let debouncedFn: ICancelable<typeof fn> & typeof fn;
+
+    beforeEach(() => {
+      jest.useFakeTimers();
+
+      callCount = 0;
+      async = new Async();
+      fn.mockClear();
```

Comment:
clear mock `afterEach`

Reply:
Moved mock and `callCount` clearing to `afterEach`. Thanks!